code/modules/mob/living/simple_animal/abnormality/he/nameless_fetus.dm

```diff
@@ -0,0 +1,104 @@
+/mob/living/simple_animal/hostile/abnormality/fetus
+	name = "Nameless Fetus"
+	desc = "A giant, pus-filled baby."
+	icon = 'ModularTegustation/Teguicons/48x48.dmi'
+	icon_state = "fetus"
+	maxHealth = 400
+	health = 400
+	threat_level = HE_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(40, 50, 60, 60, 60),
+		ABNORMALITY_WORK_INSIGHT = list(20, 30, 30, 30, 30),
+		ABNORMALITY_WORK_ATTACHMENT = list(20, 30, 30, 30, 30),
+		ABNORMALITY_WORK_REPRESSION = list(20, 30, 30, 30, 30)
+		)
+	start_qliphoth = 1
+	pixel_x = -8
+	base_pixel_x = -8
+
+	max_boxes = 16
+	work_damage_amount = 7
+	work_damage_type = RED_DAMAGE
+	ego_list = list(
+//		/datum/ego_datum/weapon/syrinx,		Will update with the rest of the dump EGO
+		/datum/ego_datum/armor/syrinx
+		)
+//	gift_type =  /datum/ego_gifts/syrinx
+	abnormality_origin = ABNORMALITY_ORIGIN_LOBOTOMY
+	var/mob/living/carbon/human/calling = null
+
+/mob/living/simple_animal/hostile/abnormality/fetus/ZeroQliphoth(mob/living/carbon/human/user)
+	check_players()
+	check_range()
+
+	//Are they nearby?
+/mob/living/simple_animal/hostile/abnormality/fetus/proc/check_range()
+	if(calling && (calling in range(1, src)))
+		var/mob/living/carbon/human/Y = calling
+		Y.gib()
+
+		for(var/mob/living/carbon/human/H in GLOB.player_list)
+			to_chat(H, "<span class='userdanger'>The fetus is satisfied.</span>")
+
+		notify_ghosts("The fetus is satisfied.", source = src, action = NOTIFY_ORBIT, header="Something Interesting!") // bless this mess
+		datum_reference.qliphoth_change(1)
+
+	addtimer(CALLBACK(src, .proc/check_range), 2 SECONDS)
+
+
+/mob/living/simple_animal/hostile/abnormality/fetus/proc/check_players()
+	if(datum_reference.qliphoth_meter == 1)
```

Comment:
This might cause problems if some other source causes it to raise above 1, so it's best to check if it's just above zero.
```suggestion
	if(datum_reference.qliphoth_meter > 0)
```

Reply:
It's actually hardcoded to where the Qliphoth meter never goes above 1. If it ever goes above 1, it's set to one.